tests/integration/models/device.spec.js

```diff
@@ -114,10 +114,12 @@ describe('Device Model', function () {
 					.to.be.rejectedWith('Invalid device type: foobar');
 			});
 
-			it('should become the manifest given a device type alias', () =>
-				balena.models.device
-					.getManifestBySlug('raspberrypi')
-					.then((manifest) => expect(manifest.slug).to.equal('raspberry-pi')));
+			it('should become the manifest given a device type alias', async () => {
+				const manifest = await balena.models.device.getManifestBySlug(
+					'raspberrypi',
+				);
+				return expect(manifest.slug).to.equal('raspberry-pi');
```

Comment:
NIT
```suggestion
				expect(manifest.slug).to.equal('raspberry-pi');
```
I see that we can also drop the `return` in a lot other tests below, now that `.eventually` got removed, but we can leave dropping those for later.

Reply:
I'm going to leave tidying up of coffeescript implicit returns, I mainly just wanted to handle the async/await of runtime code and the tests are just a bonus because it seemed relatively straightforward to do (although as it turns out the missing typings for a bunch of stuff has made it less straightforward than I initially thought)